exporter/datadogexporter/README.md

```diff
@@ -7,6 +7,8 @@ This exporter sends metric and trace data to [Datadog](https://datadoghq.com). F
 > which contains recommendations on securing sensitive information such as the
 > API key required by this exporter.
 
+> **Note**: Metrics exporting is not considered stable yet and may suffer breaking changes.
```

Comment:
Very nitpicky: I think referring to the component is slightly clearer than referring to the function:
```suggestion
> **Note**: The metrics exporter is not considered stable yet and may suffer breaking changes.
```